### DISCUSSION TITLE
How should I Add contacts?

### DISCUSSION THREAD

**Discussion by hassanAmmag** [nioc/xmpp-web#112] [Q&A] [answered]
Hey, first of all great repo! Implemented this in my react native app. Im able to complete the rooms modules successfully with my own Ejabberd server. 

Im stuck in the 1 to 1 chatting and adding contacts, im not able to see what exactly adding 2 users together in roster. I need a bit of guidance here. Thank you so much!

What Im doing is, im calling **sendMessage** function with isMuc as false to make it a "chat" but not being able to retrieve anything on the other end coz probably not in roster?

**Comment by nioc** [comment, marked as answer]
Hello, for now, you need another XMPP client to add contact in roster (both subscribe).
